public/spec/bowlingSpec.js

```diff
@@ -0,0 +1,202 @@
+describe ('Bowling', function(){
+
+  var bowling;
+
+  beforeEach(function(){
+    bowling = new Bowling();
+  });
+
+  describe('bowl1', function(){
+    it('return the score', function(){
+      expect(bowling._bowl1(3)).toEqual(3);
+    });
+    it("it return 'strike' when 10 is scored", function(){
+      expect(bowling._bowl1(10)).toEqual("strike")
+    });
+    it('changes the frameStatus to strike if a strike is bowled', function(){
+      bowling.frame(10, 0)
+      expect(bowling.frameStatus).toEqual('strike')
+    });
+  });
+
+  describe('bowl2', function(){
+    it('returns the score for bowl1 and bowl2', function(){
+      bowling._bowl1(3);
+      expect(bowling._bowl2(5)).toEqual([3, 5])
+    });
+    it('returns another score for bowl1 and bowl2', function(){
+      bowling._bowl1(5);
+      expect(bowling._bowl2(2)).toEqual([5, 2])
+    });
+    it('returns spare if the total for the first and second bowl is 10', function(){
+      bowling._bowl1(7)
+      expect(bowling._bowl2(3)).toEqual('spare')
+    });
+    it('changes the frameStatus to spare if spare is bowled', function(){
+      bowling.frame(6, 4)
+      expect(bowling.frameStatus).toEqual('spare')
+    });
+  });
+
```

Comment:
Tests are clear and grouped by which method is being tested which makes the spec file easier to follow.